libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -39,10 +39,17 @@ import {
 } from '../get-decrypted-secret-key';
 import { BRIDGE_EXECUTION_ERROR } from '../../utils';
 
-export const DEFAULT_TIMEOUT = 15_000; // 15 seconds
+export const DEFAULT_TIMEOUT = 5_000; // 5 seconds
 export const DEFAULT_RETRIES_LIMIT = 3;
 export const RETRYABLE_HTTP_CODES: number[] = [
-  408, 413, 429, 500, 502, 503, 504, 521, 522, 524,
+  408, // Request Timeout
+  429, // Too Many Requests
+  500, // Internal Server Error
+  503, // Service Unavailable
+  504, // Gateway Timeout
+  521, // CloudFlare TCP Error
+  522, // CloudFlare Web Server Connection Error
+  524, // CloudFlate Timeout Error
```

Comment:
Removed `413` and `502` as they are typically not retryable status codes ([ref](https://stackoverflow.com/questions/47680711/which-http-errors-should-never-trigger-an-automatic-retry))